client/openocd_rpc.py

```diff
@@ -141,9 +146,19 @@ def read_word(self, address):
         with self._lock:
             return self._maybe_retry(self._read_word_locked, address)
 
+    def _should_ignore(self, line):
+        line = line.decode('utf-8')
+        for regexp in self._ignore_regexps:
+            if re.match(regexp, line):
+                return True
+        return False
+
     def _read_word_locked(self, address):
         """Reads a 32-bit word from given address."""
         out = self._send_command_locked('ocd_mdw 0x%x' % address)
+        while self._should_ignore(out):
```

Comment:
Hmm. This will only catch the error if it happened with `read_word`. If it happens during `read_memory`, it will still explode. We might want to split the output at L142, and filter the lines out there. 

Or put in a TODO and we'll deal with it if we see it again.